meta-resin-common/recipes-core/packagegroups/packagegroup-resin.inc

```diff
@@ -13,4 +13,5 @@ RDEPENDS_${PN} = " \
     rsync \
     kernel-modules \
     os-release \
+    resin-device-uuid \
```

Comment:
This should not be a dependency here. resin-device-uuid is needs to be a dependency of the services which need it. openvpn, resin-supervisor and resin-register.